src/Pim/Bundle/CatalogBundle/Resources/config/completeness_checkers.yml

```diff
@@ -3,28 +3,33 @@ parameters:
     pim_catalog.completeness.checker.media.class:   Pim\Component\Catalog\Completeness\Checker\MediaCompleteChecker
     pim_catalog.completeness.checker.metric.class:  Pim\Component\Catalog\Completeness\Checker\MetricCompleteChecker
     pim_catalog.completeness.checker.price.class:   Pim\Component\Catalog\Completeness\Checker\PriceCompleteChecker
-    pim_catalog.completeness.checker.empty.class:   Pim\Component\Catalog\Completeness\Checker\EmptyChecker
+    pim_catalog.completeness.checker.simple.class:  Pim\Component\Catalog\Completeness\Checker\SimpleCompleteChecker
 
 services:
     pim_catalog.completeness.checker.chained:
```

Comment:
`Pim\Component\Catalog\Completeness\Checker\ProductValueCompleteChecker` will be the chained checker, as it's the main entry point